data-serving/scripts/export-data/common.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-export ECR=612888738066.dkr.ecr.us-east-1.amazonaws.com
+export ECR=612888738066.dkr.ecr.eu-central-1.amazonaws.com
```

Comment:
I would expect the number to be different as well. We will know when we create the new repo.

Reply:
I think the number is our AWS account which shouldn't change - but good to parametrize that as well if possible